coordinator/main.go

```diff
@@ -38,6 +38,7 @@ const (
 
 func main() {
 	if err := run(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: running coordinator: %v\n", err)
```

Comment:
I just realized that we're actually trying to log the error in the defer statement a few lines below. Any ideas why this is not working as expected?

Reply:
Ah, good point. I guess the coordinator waits for the err group goroutines to finsh. The cleanup in the shutdown goroutine doesn't have a timeout, an grpc servers aren't guaranteed to shutdown anytime.

Reply:
revert: https://github.com/edgelesssys/contrast/pull/1290